### PR TITLE
#413: Add documentation for abs op

### PR DESCRIPTION
**Pull request by thearusable**
Related to #413 

This PR adds documentation page for abs op.

[Updated] Page:
Zrzut ekranu 2022-09-26 o 12 02 06

### CODE REVIEW

File: docs/source/components/ops/abs.rst [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+  template <class T1, class T2>
+  void abs(T1 & destination, const T2 & source);
+
+  }} // end namespace pressio::ops
+
+Parameters
+----------
+
+* ``destination``: the object to store absolute value
```
Comment (fnrizzi) [inline docs/source/components/ops/abs.rst:23]: "the object to write to"

File: docs/source/components/ops/abs.rst [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+* ``source``: the object to calculate abs
```
Comment (fnrizzi) [inline docs/source/components/ops/abs.rst:25]: "the object whose elements are used to compute the absolute value of"

File: docs/source/components/ops/abs.rst [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+
+Constraints
+~~~~~~~~~~~
+
+- ``T1`` and ``T2`` must be:
+
+  - an Eigen vector, i.e. ``pressio::is_vector_eigen<T>::value`` or
+
+  - a epetra vector, i.e. ``pressio::is_vector_epetra<T>::value`` or
```
Comment (fnrizzi) [inline docs/source/components/ops/abs.rst:34]: please capitilize Epetra

File: docs/source/components/ops/abs.rst [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+
+  - a Kokkos rank-1 view, i.e. ``pressio::is_vector_kokkos<T>::value`` or
+
+  - a tpetra vector, i.e. ``pressio::is_vector_tpetra<T>::value`` or
+
+  - a tpetra block vector, i.e. ``pressio::is_vector_tpetra_block<T>::value``
```
Comment (fnrizzi) [inline docs/source/components/ops/abs.rst:40]: can you use a consistent style: Tpetra

File: docs/source/components/ops/abs.rst [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+
+Return value
+~~~~~~~~~~~~
+
+None
+
+Effects
+~~~~~~~
+
+:red:`finish`
```
Comment (fnrizzi) [inline docs/source/components/ops/abs.rst:61]: can you please try to fill in the what the function does?

**Comment by fnrizzi** [comment]
thanks! 
a few comments: 
- i think this operation also supports expressions acting on Eigen or Kokkos rank-1 objects , can you please add that?

File: docs/source/components/ops/abs.rst (hunk 1 above)
Comment (thearusable) [reply docs/source/components/ops/abs.rst:23]: Updated

File: docs/source/components/ops/abs.rst (hunk 2 above)
Comment (thearusable) [reply docs/source/components/ops/abs.rst]: Updated, thanks!

File: docs/source/components/ops/abs.rst (hunk 3 above)
Comment (thearusable) [reply docs/source/components/ops/abs.rst]: Updated

File: docs/source/components/ops/abs.rst (hunk 4 above)
Comment (thearusable) [reply docs/source/components/ops/abs.rst]: Sure, updated

File: docs/source/components/ops/abs.rst (hunk 5 above)
Comment (thearusable) [reply docs/source/components/ops/abs.rst]: Sure I will try for all of them to add something in the Return value/Effects sections

**Pull request closed without merging**